activity/activity.info

```diff
@@ -1,8 +1,8 @@
 [Activity]
 name = Frotz
 activity_version = 3
-bundle_id = vu.lux.olpc.Frotz
-exec = sugar-activity frotz.FrotzActivity
+bundle_id = org.sugarlabs.Frotz
```

Comment:
Is there any particular reason why you changed the bundle id?

Reply:
It was giving bundle_id not found error when I tested with python2. But after making changes the error has gone. The activity was not starting before with python2. You can try this one on python2 by changing. Thanks.

Reply:
Reasons not to change the _bundle_id_ include;
* when an upgrade of an activity occurs on a system, Journal objects from the old version can continue to be used on the new version; otherwise those Journal objects cannot be opened,
* when an activity is shared between a system using the old _bundle_id_ and another system using the new _bundle_id_, the sharing will work; otherwise the shared instance will not be displayed in the Neighbourhood View,
* when an upgrade happens while Sugar is running, Sugar will represent only _one_ activity by this name on the menu; otherwise Sugar may represent _both_ the old and new activities on the menu at once if there is a copy of the activity in `~/Activities` as well as `/usr/share/sugar/activities`,
* locating source code repositories is easy because we can search for the name; otherwise it becomes harder.

I don't think a _bundle_id_ not found error is a correct reason to change the _bundle_id_.  Next time that happens, try to diagnose why it happened rather than jumping to conclusions about the cause.  :grin: